mammoth/model_builder.py

```diff
@@ -314,13 +340,28 @@ def build_model(
         device = torch.device("cpu")
     logger.info(device)
 
+    enc_adapters_by_name: Optional[Dict[str, Adapter]] = build_adapters(
```

Comment:
the type hint at the var declaration is a bit much. maybe worth a comment insteaed? you already have a type hint for the return value of `build_adapters`